tasks/internal_reconcile_report_orphan/internal_reconcile_report_orphan.py

```diff
@@ -26,6 +26,13 @@
 PAGE_SIZE = 100
 
 
+with open("schemas/input.json", "r") as raw_schema:
+    schema = json.loads(raw_schema.read())
```

Comment:
I noticed schema variable is reused making it mutable. You may want to change it to `input_schema` for this instance and `output_schema` for the other instance to better clarify the code and make the input for validation more immutable.

Reply:
changed